docs/Users/Tutorials/Testing_Bears.rst

```diff
@@ -0,0 +1,196 @@
+How to use LocalBearTestHelper to test your bears
+=================================================
+
+coala has an awesome testing framework so that you can write tests for bears
```

Comment:
imo i'd bold coala everywhere, so it gets out of the normal, so **coala**, but this is just an opinion.